aiohttp/protocol.py

```diff
@@ -173,7 +175,7 @@ def __call__(self, out, buf):
             raw_data = yield from buf.readuntil(
                 b'\r\n\r\n', self.max_headers)
         except errors.LineLimitExceededParserError as exc:
-            raise errors.LineTooLong(exc.limit) from None
+            raise errors.LineTooLong('request header', exc.limit) from None
```

Comment:
this looks inconvenient: `LineTooLong` above is raised with single argument (formated string) and with two args here